packages/experiments/src/components/Stack/Stack.tsx

```diff
@@ -27,10 +28,16 @@ const view = (props: IViewComponentProps<IStackProps, IStackStyles>) => {
         const stackItemFirstChildren = React.Children.toArray(children) as React.ReactElement<{ className?: string }>[];
         const stackItemFirstChild = stackItemFirstChildren && stackItemFirstChildren[0];
 
+        // pass down both the className on the StackItem as well as the className on its first child
+        let mergedClassName = defaultItemProps.className;
+        if (stackItemFirstChild && stackItemFirstChild.props && stackItemFirstChild.props.className) {
+          mergedClassName = mergeStyles(mergedClassName, stackItemFirstChild.props.className);
+        }
+
         return React.cloneElement(child, {
           ...defaultItemProps,
           ...child.props,
-          className: stackItemFirstChild && stackItemFirstChild.props ? stackItemFirstChild.props.className : undefined
+          className: mergedClassName
```

Comment:
I understand what's going on here, but I think perhaps you don't need to call `mergeStyles`. What you're trying to do is:

If they have a `className` being provided, pass that through. But also, provide the styling applicable to the stack item itself.

You could just use the css class name appending:

```
css(defaultItemProps.className, stackItemFirstChild.props.className)
```

which is a glorified array join with special consideration of ignoring falsey values.

Internally mergeStyleSets on the child will auto expand the rules and do the right merging.